app/js/features/buyer/reducer.js

```diff
@@ -0,0 +1,32 @@
+import {SET_CONTACT, SET_OFFER_ID, SET_TRADE} from './constants';
+
+const DEFAULT_STATE = {};
+
+function reducer(state = DEFAULT_STATE, action) {
+  switch (action.type) {
+    case SET_CONTACT:
+      return {
+        ...state, ...{
+          nickname: action.nickname,
```

Comment:
Is it really buyer? I would have imagine all of that is part of the profile?
I think we have some overlap between buy and sell and that should be only one.

Reply:
I'll wait until refactoring those, because I'm still unsure how we handle it. In the end, we might even just use the Status nickname, so no need for the form.

Depends also if we do save the name and of the contact code in the contract. If yes, having them inside buyer simplifies the gathering of the information.

Anyway, we'll see.